docs/run.md

```diff
@@ -0,0 +1,21 @@
+# Running the simulation
+
+The two basic ways to run the simulation are a Python package and command line interface. Both interfaces take simulation parameters as a configuration file and several other user inputs as function arguments or command line arguments.
+
+The configuration file contains all parameters needed for a single simulation run. The package documentation explains the configuration parameters in detail. The configuration format is YAML which is both human and machine readable and writable and JSON is supported as an alternative. The configuration file consists of a set of key and value pairs, which specify the parameters used for consignment generation, contamination, and inspection.
+The path to the configuration file must be provided as a function or command line argument, along with the number consignments and stochastic runs to simulate.
+
+You can use the package from either the command line or Python.
```

Comment:
I'm not clear on what you mean by "Python" here. Isn't it also run with Python in the CL?

Reply:
Right, Python is used in both cases. What I mean is in CL you interact using the CLI of the package. You don't leave your command line (you would interact the same if the code would be in R or anything else). In Python (script, shell, notebook) you interact using Python API. Perhaps something like:

_You can use the package from the command line (using the command line interface of the Python package) or from directly Python._

Let me know if you have some other suggestions.

Reply:
Ok, this is still a little fuzzy for me. Would it make sense to say this?

_You can use the package from the command line (using the command line interface of the Python package) or  directly from Python (within an IDE or Jupyter notebook)._

Reply:
Yes. This makes sense. I used this sentence. I think the distinctions is between a mode of interaction and what runs in the background. Your sentence gives context to what "use" means here, so that is hopefully enough.